app/api/routes.py

```diff
@@ -144,5 +144,21 @@ def update_watering():
         'next_water': next_water
     }
     response = jsonify(results)
+    response.status_code = 201
+    return response
+
+@api.route('garden/plants/<int:id>')
+def get_gardenplant(id):
+    gardenplant = GardenPlant.query.get_or_404(id)
```

Comment:
I think we will want to reverse engineer this so that we don't have to pass in a gardenplant ID with this route. Same for the watering endpoint